client/dashboard/task-list/style.scss

```
@@ -182,7 +184,8 @@
 		position: relative;
 
 		.muriel-input-text input {
-			left: $gap * 2;
+			left: $gap;
+			max-width: 90%;
```

Minor nitpick: probably won't affect any screen sizes, but we can change to the following to fill this area snuggly:

```suggestion
			max-width: calc(100% - #{$gap});
```